assets/style/_header.scss

```
@@ -1,28 +1,32 @@
 header {
-  height: 32.5rem;
+
+  h1 {
+    font-family: 'Prestafont';
+    font-size: 95px;
+  }
 
   .gradient {
     width: 100%;
     height: 100%;
     background: $primary;
   }
 
-  .white-text {
+  .black-text {
+    font-family: 'IBM Plex Sans', sans-serif;
```

```suggestion
    font-family: 'IBM Plex Sans', Arial, sans-serif;
```
And same for all other places with IBM Plex Sans :D

---

Fixed in https://github.com/PrestaShop/TopContributors/pull/118/commits/ccb67fff2bd015ae2c21aa81fdc391f713eaf909